COMMUNITY.md

```diff
@@ -1,6 +1,6 @@
 # OpenSDS Controller Work Group
 
-Covers discussion of OpenSDS Controller architecture design and featrue development.
+Covers discussion of OpenSDS Controller architecture design and feature development.
```

Comment:
Why do you include the documentation change as part of this profiles design PR? Is this a rebase issue?  If you made these changes here, can you separate them into a different PR?

Reply:
I just wanted to fix this typo error, it's ok to separate it into another PR.